### ISSUE TITLE
Pedestrian Recall feature layer publishing - B. Craig

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#2559]
Email from Brian:

> I’ve been working on a map for Ped Recall for Jen and Lewis. Most of my work consists on the back end running some SQL queries and some analysis in Excel. I joined an excel table to a signals GIS layer that I exported from arc online. What would the best way of getting this online be? How difficult is it to update the map at that point. My schedule is up-to-date if you want to meet to discuss this.

**Comment by jaime-mckeown** [comment]
- Brian and I discussed what he needs from the data. He has already created the shapefile from the Signal feature layer and the Spreadsheet he exported. The map he gave me has the same data symbolized 3 different ways. I will publish the feature layer with the data setup the same way so it's easier for people to use. Share with Organization, AMD, VZ. He will need it updated periodically, but automation is not needed as it should not be very often.
- Saving work to G:\ATD\ATD_GIS\Arterial_Management\One_offs\Pedestrian_Recall\

**Comment by jaime-mckeown** [comment]
- Since Brian exported the data to a shapefile, the field names were cut off. I imported it to a FGDB and fixed the field names, then exported on Turned-on signals for the layer to use in the feature layer.
- Imported the Excel Spreadsheet to the FGDB and added a text field for Signal ID for joining.
- Created a new Mxd with the 3 different layers Brian had in his version based on the data I created. 
- Published a new feature layer to AGOL, shared with the Organization, AMD and VZ.
- Emailed Brian with a link to the new feature layer.
